src/middleware/github-webhook-middleware.ts

```diff
@@ -116,7 +116,7 @@ export const GithubWebhookMiddleware = (
 			fields: {
 				webhookId,
 				gitHubInstallationId,
-				gitHubServerAppIdPk: "" + gitHubAppId,
+				gitHubServerAppIdPk: gitHubAppId?.toString(),
```

Comment:
Hmm... isn't this changes the behaviour for undefined? Before the change it would be "undefined" as a string, after the change it would be undefined as a value... If so then wondering which one of those is the behaviour we want?